### ISSUE TITLE
label should have a disabled state

### ISSUE THREAD

**Issue by khamudom** [Microsoft/fast-dna#980]
the label component for fast component msft react should have a disable prop which causes the label to match the disabled input.

**Comment by nicholasrice** [comment]
I don't think we want this - a text element has no concept of being "disabled" because it isn't an interactive component.
